main.py

```diff
@@ -51,16 +50,21 @@ def aggregate_seperated_store_files() -> pd.DataFrame:
     return stores_df
 
 def read_housing_data() -> pd.DataFrame:
-    return csv_to_df(HOUSING_PRICES_FILE)
+    url = 'https://raw.githubusercontent.com/ChaseCallahan37/541_group_proj/main/housing-data/realtor-data.csv'
```

Comment:
Lets make this a constant global variable up top with the others. something like:
`REALTOR_DATA_URL`